frontend/src/components/Pokemon/PokemonDetail/PokemonDetail.tsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from 'react'
+import Loader from 'components/Loader/Loader'
+import ErrorCard from 'components/Error/Error'
+import styles from './PokemonDetail.module.css'
+import { fetchPokemons } from '../data_integration/fetchPokemons'
+import { PokemonProps } from '../domain/buisness_objetcs/pokemon.type'
+
+const PokemonDetail = ({ pokeId }: { pokeId: string | undefined }) => {
+   const [pokemonDetails, setPokemonDetails] = useState<PokemonProps>({
+      name: '',
+      id: 0,
+      weight: 0,
+      height: 0,
+   })
+   const [isLoading, setIsLoading] = useState(true)
+   const [errorMessage, setErrorMessage] = useState('')
+
+   const updatePokemonDetails = async () => {
+      const pokemonDetails = await fetchPokemons(`http://localhost:8000/pokemon/${pokeId}`)
+      setIsLoading(false)
```

Comment:
- <!-- TODO's are small, trivial, but necessary changes. Distinguishing todo comments from issues: or suggestions: helps direct the reader's attention to comments requiring more involvement. -->
    
    **🧹 todo:** <subject>No magic number / string. Set this in a variable (data_integration)
    
    <discussion>